index.html

```diff
@@ -358,13 +358,148 @@ <h2 class="page-section-heading text-center text-uppercase text-secondary mb-3">
                     Campus Life
                 </h2>
                 <p class="lead text-center text-secondary mb-0">
-                    Find amenities on campus and come prepared for Room Draw.
+                    Locate amenities & lost items, come prepared for Room Draw, and stay active.
```

Comment:
On line 607, the "&" is used in place of the "and" after the comma so I think we should keep it consistent here:
> Locate amenities and lost items, come prepared for Room Draw, & stay active.